Populate ack (#470)
usegalaxy-eu/website: citations.md

# Galaxy EU citations

<br>

> This is the list of scientific publications that cited the European Galaxy instance sorted by year. For acknowledgement, please refer to the [**About section**](/about).

Reviewer: maybe more active. Please help us fund this project by .... ?

> This is the list of scientific publications that cited the European Galaxy instance sorted by year. For acknowledgement, please refer to the [**About section**](/about).

Author: ```suggestion
This is the list of scientific publications that cited the European Galaxy instance sorted by year. Please help us  to support this project with your [acknowledgement](/about).
```
